ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -380,17 +372,20 @@ TFLiteInterpreter::getTensorDim (int tensor_idx, tensor_dim dim)
  */
 int
 TFLiteInterpreter::setTensorProp (const std::vector<int> &tensor_idx_list,
-    GstTensorsInfo * tensorMeta)
+    GstTensorsInfo * tensorMeta, std::vector <TfLiteTensor *>& tensor_ptrs)
```

Comment:
It seems that `tensors_ptrs` is a member variable of `TFLiteInterpreter`. Why this should be passed as an argument?

Reply:
This function is commonly used for both inputs and outputs.
Hence tensor_ptrs is passed which is different for input or output.

Reply:
Ah, okay